### PR TITLE
updated React-to-messages 

### PR DESCRIPTION
**Pull request by ayrotideysarkar**

### CODE REVIEW

**Comment by ayrotideysarkar** [comment]
can i see a preview?
@cwarnermm and @justinegeffen

File: source/channels/react-to-messages.rst [hunk 1]
```diff
@@ -4,64 +4,84 @@ React to messages using emojis
...
+     3. If required, use the Search bar to find your custom emoji in the list.
+     4. Under **Actions** select **Delete**.
+     5. Choose **Delete** to confirm.
+
+       .. image:: ../images/delete_custom_emoji.png
+           :alt: Remove custom emoji
+      
+  .. tab:: Mobile app
+   
+    **To react to messages with emoji in the mobile app v2**
```
Comment (cwarnermm) [inline source/channels/react-to-messages.rst:72]: Please remove references to mobile v2 for now. We'll incorporate needed references to v2 as we get closer to v2 being released.

File: source/channels/react-to-messages.rst [hunk 2]
```diff
@@ -4,64 +4,84 @@ React to messages using emojis
...
+
+     Select the **Smile** icon |smile-icon| inside the Mattermost message input box to open the emoji picker.
+
+    **Recent emojis**
+    
+     Tap on the **Smile** icon |smile-icon| to select recent emojis.
+                
+    **Select default skin tone in the mobile app**
+
+     Mobile v2 doesn't currently support skin tone selection.
```
Comment (cwarnermm) [inline source/channels/react-to-messages.rst:82]: Another v2 reference to remove

**Review by cwarnermm — COMMENTED** [review]
Thanks, @ayrotideysarkar! I've provided feedback for you inline.

**Review by cwarnermm — APPROVED** [review]
Thank you, @ayrotideysarkar!

**Pull request merged**